source/_posts/2017-05-19 kanban-method

```diff
@@ -0,0 +1,74 @@
+# Kanban method in a consulting company
+
+
+
+Before we start we need to know what is that Kanban. Let me show you a short explanation.
+
+> **Kanban** is a method for managing knowledge work which balances demands for work with the available capacity for new work. 
```

Comment:
Please reconsider this sentence: `knowledge work which balances demands for work...`.